x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -531,6 +533,166 @@ class AgentPolicyService {
     };
   }
 
+  public async bulkUpdate(
+    soClient: SavedObjectsClientContract,
+    esClient: ElasticsearchClient,
+    agentPolicies: Array<Partial<AgentPolicy> & { id: string }>,
+    options: {
+      user?: AuthenticatedUser;
+      force?: boolean;
+      skipValidation?: boolean;
```

Comment:
This option seems dangerous. So this externally exposed fleet service is allowing a caller to disabled validations? 🤔 

Also - if you decide to still keep it, what validations are skipped? from the code below, looks like only output validations. maybe add a comment indicating that